load_easybuild_module.sh

```diff
@@ -128,4 +128,8 @@ else
     fatal_error "EasyBuild not working?!"
 fi
 
+# EESSI-extend checks for the EB version being used.
+# If EESSI-extend is already loaded, we need to reload it in order to reevaluate the checks.
+module is-loaded EESSI-extend && module update
```

Comment:
This doesn't control the order in which modules are reloaded, so may not fix anything.

I would `unload` and then `load` EESSI-extend?

Reply:
Just doing a m`module load EESSI-extend` again (if it's loaded) should work:
```
$ source /cvmfs/software.eessi.io/versions/2023.06/init/bash
$ ml EESSI-extend

{EESSI 2023.06} $ echo $EASYBUILD_CUDA_SANITY_CHECK_ERROR_ON_FAILED_CHECKS 
1
{EESSI 2023.06} $ ml EasyBuild/4.9.4

The following have been reloaded with a version change:
  1) EasyBuild/5.1.1 => EasyBuild/4.9.4

{EESSI 2023.06} $ echo $EASYBUILD_CUDA_SANITY_CHECK_ERROR_ON_FAILED_CHECKS 
1
{EESSI 2023.06} $ ml EESSI-extend
{EESSI 2023.06} $ echo $EASYBUILD_CUDA_SANITY_CHECK_ERROR_ON_FAILED_CHECKS 
```